src/content/en/fundamentals/primers/shadowdom/index.markdown

```diff
@@ -984,8 +984,11 @@ function loadScript(src) {
 
 // Lazy load the polyfill if necessary.
 if (!supportsShadowDOMV1) {
-  loadScript('/bower_components/shadydom/shadydom.min.js');
-  loadScript('/bower_components/shadycss/shadycss.min.js');
+  loadScript('/bower_components/shadydom/shadydom.min.js')
+    .then(loadScript('/bower_components/shadycss/shadycss.min.js'))
```

Comment:
You need to wrap this in a function so that the promises execute in the proper order:

```js
.then(() => loadScript('/bower_components/shadycss/shadycss.min.js'))
```

Or if you don't care about the loading order, do

```js
Promise.all([
  loadScript('/bower_components/shadydom/shadydom.min.js'),
  loadScript('/bower_components/shadycss/shadycss.min.js')
]).then(() => {
  // Polyfills loaded.
});
```